motor_control/Triple_HBridge/3PhaseMotorControl_IR2104.ino

```diff
@@ -0,0 +1,158 @@
+#include <ramping.h>

```

Comment:
This code doesn't use ramping implementation from the ramping.h header file. This integrated one might be the other one

Reply:
Looking at the code it seems like both branches use the ramping.h objects in that they create a camping object and call its member functions for finding new speeds. Is there some other difference between the two implementations that I did not know about

Reply:
We started working with the ramping file only once I'd been working with the UCC227 chip, and I forgot to implement the ramping into the code for the original chip. I just updated it though.